compose/ui/ui/src/webCommonW3C/kotlin/androidx/compose/ui/window/ComposeWindow.js.kt

```diff
@@ -219,7 +222,13 @@ private class ComposeWindow(
             event.preventDefault()
         })
 
+        val setOfClipboardKeys = setOf(SkikoKey.KEY_V, SkikoKey.KEY_C, SkikoKey.KEY_X)
         addTypedEvent<KeyboardEvent>("keydown") { event ->
+            val skikoEvent: SkikoKeyboardEvent = event.toSkikoEvent(SkikoKeyboardEventKind.DOWN)
+            if (skikoEvent.key in setOfClipboardKeys && skikoEvent.modifiers != SkikoInputModifiers.EMPTY) {
```

Comment:
To things I wanted to discuss regarding this fragment. First, we already have a mechanism for passing events to the browser - that's `onKeyboardEventWithResult` what if we'll move this logic of whether we are dispatching clipboard event over there? 

Second, it seems to be the right way to filter out some specific keyboard event is to have a helper function that have actual access to the keymappings . Check out, for instance, this code
```
internal actual fun isCopyKeyEvent(keyEvent: KeyEvent) =
    platformDefaultKeyMapping.map(keyEvent) == KeyCommand.COPY
```